### PR TITLE
PRs only by nathan streak: 7

### PR DESCRIPTION
**Pull request by NathanCrazy**

### CODE REVIEW

**Comment by NathanCrazy** [comment]
did it work

**Comment by milenakos** [comment]
i didnt test it

**Comment by milenakos** [comment]
i mean we can just test it in prod

**Comment by NathanCrazy** [comment]
yooo thats my favorite thing to do

**Comment by NathanCrazy** [comment]
testing in prod my beloved

File: main.py [hunk 1]
```diff
@@ -2810,12 +2821,12 @@ async def lb_handler(interaction, type, do_edit=None):
                 messager = round(messager / 3600, 2)
 
         # dont show placements if they arent defined
-        if interactor and (type == "Cats" or type == "Slowest") and interactor <= 0:
+        if interactor and (type == "Cats" or type == "Slowest"or type == "Value") and interactor <= 0:
```
Comment (milenakos) [inline main.py:2824]: buddy ate the space

Comment (NathanCrazy) [reply main.py:2824]: okay fine ill make it consistent

Comment (NathanCrazy) [reply main.py:2824]: if(interactor)and(type=="Cats"or(type)=="Slowest"or(type)=="Value")

**Comment by NathanCrazy** [comment]
merge so i can test pls 🥺

**Comment by milenakos** [comment]
im like 95% sure its broken so i dont really want to merge this without testing

**Comment by NathanCrazy** [comment]
damn ok
u gonna put it on cat bot 2 then?

**Comment by milenakos** [comment]
wonderful it literally syntax errors

**Comment by milenakos** [comment]
there is 8 editor errors on one line wtf did you do :sob:

**Comment by NathanCrazy** [comment]
uhhhh which one :3

File: main.py [hunk 2]
```diff
@@ -2772,6 +2774,15 @@ async def lb_handler(interaction, type, do_edit=None):
                 if len(rarest_holder) > 10:
                     joined = f"{len(rarest_holder)} people"
                 string = f"Rarest cat: {catmoji} ({joined}'s)\n"
+        elif type == "Value":
+            unit = "value"
+            total_sum_expr = peewee.fn.SUM(sum((len(CAT_TYPES) / type_dict[cat_type]) * getattr(Profile, f"cat_{cat_type}").cast("BIGINT")] for cat_type in cattypes))
```
Comment (milenakos) [inline main.py:2779]:
```suggestion
            total_sum_expr = peewee.fn.SUM(sum((len(CAT_TYPES) / type_dict[cat_type]) * getattr(Profile, f"cat_{cat_type}").cast("BIGINT") for cat_type in cattypes))
```

**Comment by milenakos** [comment]
in other news it works although the value isnt rounded so it looks kinda weird

**Comment by NathanCrazy** [comment]
oh

**Comment by NathanCrazy** [comment]
well thats good news

**Comment by milenakos** [comment]
i think rounding just makes sense to make it consistent with all other value displays

**Comment by NathanCrazy** [comment]
yeah definitely

**Comment by milenakos** [comment]
no i think in display would make more sense for speed and also for slightly more correct sorting?

**Review by milenakos — APPROVED** [review]
its 3 am i was staying awake just to merge this so good night

**Pull request merged**